### PR TITLE
Team4 question12

### PR DESCRIPTION
**Pull request by Cory-Wiard**
We answered the question about when a team will know they are finished with implementation of software project. We focused on three groups that were compiling and running the program, meeting all checks, and following the assessment and conduct guides.

### CODE REVIEW

**Comment by schultzh** [comment]
There are currently merge conflict in this PR. I would suggest merging master into your branch to get those updates.

File: management.md [hunk 1]
```diff
@@ -126,5 +126,14 @@
...
-  implementation of the software project?
+- How will your team know when it is finished with the implementation of the
+  software project?
+
+  Our team will know when it is finished with the implementation of the software
+  project because of three reasons. First, our software project will not only
+  compile, but it should run effectively with the desired output. Secondly, it
+  should pass any requirement checks for what the program should do that are
+  given at the beginning of the project. The final thing the team should do to
+  know they have finished the implementation of the project by running through
```
Comment (corlettim) [inline management.md]: "is by running through" not just "by running through"

File: management.md [hunk 2]
```diff
@@ -126,5 +126,14 @@
     the help of test cases which can indicate whether the issue(s) have been
     solved or they are still present.
 
-- How will your team know when it is finished with the
-  implementation of the software project?
+- How will your team know when it is finished with the implementation of the
+  software project?
+
+  Our team will know when it is finished with the implementation of the software
+  project because of three reasons. First, our software project will not only
```
Comment (schultzh) [inline management.md]: "because of three reasons" phrasing is very basic - maybe say something like: "when the following is complete"

**Review by schultzh — CHANGES_REQUESTED** [review]

File: management.md [hunk 3]
```diff
@@ -154,5 +154,14 @@
...
-- How will your team know when it is finished with the
-  implementation of the software project?
+- How will your team know when it is finished with the implementation of the
+  software project?
+
+  Our team will know when it is finished with the implementation of the software
+  project when the following is complete. First, our software project will not
+  only compile, but it should run effectively with the desired output. Secondly,
+  it should pass any requirement checks for what the program should do that are
+  given at the beginning of the project. The final thing the team should do to
```
Comment (corlettim) [inline management.md]: "task" instead of thing

**Review by corlettim — APPROVED** [review]

**Review by enpuyou — APPROVED** [review]

**Review by noorbuchi — APPROVED** [review]

**Review by schultzh — APPROVED** [review]
Looks good!

**Review by lussierc — APPROVED** [review]
The additions in this PR provided valuable insights to answer the question.

**Pull request merged**